pandas/core/base.py

```diff
@@ -990,11 +990,17 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
-        """ np.ndarray searchsorted compat """
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
just an FYI

we have a complete opposite policy as compared to numpy 

we go to great lengths to avoid duplication of code (doc strings whatever)

I find that just leads to s great many inconsistencies (the duplication)